doc/changes/changes_1.5.6.md

```diff
@@ -0,0 +1,30 @@
+# Row Level Security lua 1.5.6, released 2024-??-??
+
+Code name:
+
+## Summary
+
+## Features
+
+* ISSUE_NUMBER: description
```

Comment:
If you don't have a ticket, you can use the PR number here.

Reply:
That won't work, PK only accepts issues ;)

Reply:
other PR did write "n/a" here